test/integration/scheduler_perf/scheduler_test.go

```diff
@@ -153,14 +151,11 @@ func schedulePods(config *testConfig) int32 {
 
 	// Now that scheduling has started, lets start taking the pulse on how many pods are happening per second.
 	for {
-		// This can potentially affect performance of scheduler, since List() is done under mutex.
-		// Listing 10000 pods is an expensive operation, so running it frequently may impact scheduler.
 		// TODO: Setup watch on apiserver and wait until all pods scheduled.
-		scheduled, err := getScheduledPods(podLister)
+		scheduled, err := getScheduledPods(config.clientset)
```

Comment:
getScheduledPods() should accecpt `config.podInformer` or `config.podLister`

Reply:
done